ci/jobs/satellite6-python-trigger.yaml

```diff
@@ -0,0 +1,88 @@
+- job:
+    name: satellite6-pulp-trigger
+    node: f23-vanilla-np
+    wrappers:
+        - ansicolor
+    builders:
+        - shining-panda:
+            build-environment: virtualenv
+            python-version: System-CPython-2.7
+            system-site-packages: true
+            nature: python
+            command: |
+                from __future__ import print_function
+                import logging
+                import os
+                import pip
+                import pprint
+                import sys
+                import time
+                
+                pip.main(['install', 'requests'])
+                import requests
+
+                print('Print out all environment variables...')
+                pprint.pprint(os.environ)
+                JENKINS_URL = os.environ.get('REMOTE_JENKINS_URL')
+                JENKINS_API_TOKEN = os.environ.get('REMOTE_JENKINS_API_TOKEN')
+                JENKINS_USERNAME = os.environ.get('REMOTE_JENKINS_USERNAME')
+                # Change the remote job name here.
+                trigger_type = os.environ.get('TRIGGER_TYPE')
+                remote_job_name = 'elyezer-sandbox'
+                # Note: if trigger_type is 'installation'? False
+                if trigger_type == 'installation':
+                    remote_job_name = 'satellite6-installer'
+                elif trigger_type == 'robottelo':
+                    remote_job_name = 'satellite6-standalone-automation'
+
```

Comment:
What if this receive an unexpected trigger_type value? Add an else block and call sys.exit(1) to fail the job.